app/controllers/activities_controller.rb

```diff
@@ -4,6 +4,23 @@
 class ActivitiesController < AuthenticatedController
   include ProjectScoped
 
+  def index
+    @activities_groups = Activity.includes(:trackable)
```

Comment:
Let's split this into two lines instead of chaining the `group_by`:

```
activities = Activity....
@activities_groups = activities.group_by do |activity|
...
end
```